src/dev/build/tasks/install_chromium.ts

```diff
@@ -6,9 +6,11 @@
  * Side Public License, v 1.
  */
 
+import type { Logger } from '@kbn/core/server';
```

Comment:
Converted this file to TypeScript so that looking up references of our `install` function would work.

Reply:
Context on this?  Is the build logger not sufficient for `install` anymore?

Not blocking, sounds like it'll get imported either way so not worried.